tests/EdgeToEdge/Routes/ShowDonationConfirmationRouteTest.php

```diff
@@ -125,23 +101,48 @@ public function testGivenAlternativeConfirmationPageConfig_alternativeContentIsD
 			);
 			$donation = $this->newStoredDonation( $factory );
 
-			$client->request(
-				'GET',
-				'show-donation-confirmation',
-				[
-					'donationId' => $donation->getId(),
-					'accessToken' => self::CORRECT_ACCESS_TOKEN,
-					'updateToken' => self::SOME_UPDATE_TOKEN
-				]
-			);
+			$responseContent = $this->retrieveDonationConfirmation( $client, $donation->getId() );
 
-			$content = $client->getResponse()->getContent();
-			$this->assertContains( 'Alternative content', $content );
-			$this->assertContains( 'Template Name: DonationConfirmationAlternative.twig', $content );
-			$this->assertContains( 'Template Campaign: example', $content );
+			$this->assertContains( 'Alternative content', $responseContent );
+			$this->assertContains( 'Template Name: DonationConfirmationAlternative.twig', $responseContent );
+			$this->assertContains( 'Template Campaign: example', $responseContent );
 		} );
 	}
 
+	public function testGivenAnonymousDonation_confirmationPageReflectsThat() {
+		$this->createEnvironment( [], function ( Client $client, FunFunFactory $factory ) {
+			$donation = $this->newBookedAnonymousPayPalDonation( $factory );
+
+			$responseContent = $this->retrieveDonationConfirmation( $client, $donation->getId() );
+
+			$this->assertContains( 'Anonym', $responseContent );
+		} );
+	}
+
+	public function testGivenAnonymousDonation_confirmationPageShowsStatusText() {
+		$this->createEnvironment( [], function ( Client $client, FunFunFactory $factory ) {
+			$donation = $this->newBookedAnonymousPayPalDonation( $factory );
+
+			$responseContent = $this->retrieveDonationConfirmation( $client, $donation->getId() );
+
+			$this->assertContains( 'Ihre Spende per PayPal wurde gebucht', $responseContent );
+		} );
+	}
+
+	private function retrieveDonationConfirmation( Client $client, int $donationId ) {
```

Comment:
I'd have put this one after the first method using it, so less vertical navigation is needed when reading that one.